### DISCUSSION TITLE
animateTransform

### DISCUSSION THREAD

**Discussion by buryndin** [fabricjs/fabric.js#7439] [Q&A]
Does fabric support svg animation? For example https://developer.mozilla.org/en-US/docs/Web/SVG/Element/animateTransform

**Comment by ShaMan123** [comment]
You should load the svg into fabric and then use the animation mixin

**Reply by buryndin** [reply]
Can you provide an example?

**Reply by ShaMan123** [reply]
http://fabricjs.com/svg-caching

**Reply by buryndin** [reply]
No, this example rotates whole svg, but my answer about animation inside svg, please see MDN link

**Reply by ShaMan123** [reply]
fabric populates objects from an svg and groups them if necessary. I think that should answer the issue.  Elsewise, I don't understand the use case.

**Reply by buryndin** [reply]
https://codepen.io/nikhil8krishnan/pen/rVoXJa
svg can have animation inside

**Reply by ShaMan123** [reply]
Cool feature. Don't think fabric can handle this out of the box. But I don't know and can't help.

**Reply by asturur** [reply]
we wanted to parse animation from SVG, but we need first a place to store them.
I have an open PR with a concept of an animation object for the objects.
But i did not work on parsing yet https://github.com/fabricjs/fabric.js/pull/6390
